### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oliver-sanders**
* Remove Cylc 7 section from the README.
* Add a quickstart with a simple example.

Thoughts, suggestions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,42 +20,42 @@ that specialises in cycling workflows and has strong scaling characteristics.
...
+conda install cylc-flow
+
+# write your first workflow
+mkdir -p ~/cylc-src/example
+cat > ~/cylc-src/example/flow.cylc <<__CONFIG__
+[scheduling]
+    initial cycle point = 1
+    cycling mode = integer
+    [[graph]]
+        P1 = a => b[-P1] => c & d
```
Comment (MetRonnie) [inline README.md:40]: `b` isn't valid here?

**Review by MetRonnie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (oliver-sanders) [inline README.md:40]:
```suggestion
        P1 = """
            a => b => c & d
            b[-P1] => b
```

**Review by hjoliver — COMMENTED** [review]
Looks good.

I'd suggest changing the description slightly to match that on the web site:

> Cylc is a general purpose workflow engine that also orchestrates cycling systems very efficiently. It is used in production weather, climate, and environmental forecasting on HPC, but is not specialized to those domains. 

(The "also" bit is intended to allay concerns that you can only use Cylc for cycling workflows).

File: README.md [hunk 2]
```diff
@@ -20,42 +20,45 @@ that specialises in cycling workflows and has strong scaling characteristics.
 Cylc was originally developed to meet the challenges of production weather
 forecasting - which is notorious for the size and complexity of its workflows.
```
Comment (oliver-sanders) [inline README.md:21]:
```suggestion
Cylc is a general purpose workflow engine that also orchestrates cycling systems very efficiently.
It is used in production weather, climate, and environmental forecasting on HPC, but is
not specialized to those domains.
```

File: README.md [hunk 3]
```diff
@@ -17,45 +17,49 @@
 Cylc (pronounced silk) is a general purpose workflow engine
 that specialises in cycling workflows and has strong scaling characteristics.
 
-Cylc was originally developed to meet the challenges of production weather
-forecasting - which is notorious for the size and complexity of its workflows.
+Cylc is a general purpose workflow engine that also orchestrates cycling systems very efficiently.
+It is used in production weather, climate, and environmental forecasting on HPC, but is
+not specialized to those domains.
```
Comment (MetRonnie) [inline README.md:22]: A bit of repetition in these two paragraphs

Comment (hjoliver) [reply README.md:22]: Better to replace the 1st with the 2nd (maybe keep "pronounced silk" though).

Comment (hjoliver) [reply README.md]: I pushed a commit to do this.

**Review by hjoliver — APPROVED** [review]

**Pull request merged**